app/src/main/java/com/kickstarter/viewmodels/DiscoveryViewModel.java

```diff
@@ -214,7 +214,7 @@ public DiscoveryViewModel(final @NonNull Environment environment) {
       .take(1)
       .map(Intent::getAction)
       .filter(Intent.ACTION_MAIN::equals)
-      .map(__ -> DiscoveryParams.builder().staffPicks(true).build());
+      .map(__ -> DiscoveryParams.builder().build());
```

Comment:
yes on altering functionality--on ios we now default to All Projects rather than PWL. good call with the onboarding bug! will address